test/js/css-variable-reference-value.js

```diff
@@ -0,0 +1,16 @@
+suite('CSSVariableReferenceValue', function() {
+    test("The 'variable' attribute is correct", function() {
+        var expected = 'anything';
+        var temp = new CSSVariableReferenceValue(expected);
```

Comment:
Let's call this variable "referenceValue" or "result" or something like that. It's not really a temporary thing in the scheme of the test.